data/vaults/0x03403154afc09Ce8e44C3B185C82C6aD5f86b9ab.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "vault",
+  "name": "crvAAVE",
+  "description": "yearn Curve.fi aDAI/aUSDC/aUSDT",
+  "promoted": true,
+  "retired": false,
+  "migrated": false,
+  "depositDisabled": false,
+  "depositDisabledMessage": "",
+  "withdrawDisabled": false,
+  "withdrawDisabledMessage": false,
+  "apy": { "calculation": "" }
```

Comment:
@nymmrx do you know what calculation should be for this? Where are you pulling this information today? Also do we need an attribute that describes which harvest number we want to use for APY calculations? Should we add the ability to disable APY calculations?